### ISSUE TITLE
Besides using google, can I use Bing?

### ISSUE THREAD

**Issue by title51130** [NyashniyVladya/Translator3000#4]
Besides using google, can I use Bing?

---------
Thank you for program.
I've used many types of programs but it didn't work, until I found your program, it's amazing it works
thank you very much.

**Comment by NyashniyVladya** [comment]
> Besides using google, can I use Bing?

There is no such possibility at the moment. But I have already written a preparatory base for the possibility of using various translation services. This will be possible in the future.

**Comment by Feans** [comment]
Неплохо бы прикрутить яндекс или deepl. Яндекс в последнее время обогнал гугл по качеству перевода, по крайней мере в направлении англ<->русский.

**Comment by dead23angel** [comment]
> Неплохо бы прикрутить яндекс или deepl. Яндекс в последнее время обогнал гугл по качеству перевода, по крайней мере в направлении англ<->русский.

Да, только вы забыли упомянуть о том, что Яндекс убрал бесплатные апи ключи и перевод теперь платный

**Comment by Feans** [comment]
> Да, только вы забыли упомянуть о том, что Яндекс убрал бесплатные апи ключи и перевод теперь платный

Это не мешает его использовать бесплатно, обходя API. По этому же принципу если не ошибаюсь и работает Translator3000 с google translate, маскируясь под браузер. Или нет? В любом случае многие проекты работают не через API, а через имитацию chrome.

Бесплатный Qtranslate работает с яндекс.
